src/ECS/Systems/PathfindingSystem.cpp

```diff
@@ -28,43 +39,557 @@ PathfindingSystem& PathfindingSystem::instance()
 
 PathfindingSystem::PathfindingSystem() = default;
 
+namespace
+{
+
+void InitializeStep(Transform& transform, WallHug& wallHug, float angle)
+{
+	transform.rotation = glm::eulerAngleY(-angle - glm::radians(90.0f));
```

Comment:
magic number? why's it offset to 90deg?

Reply:
Not just offset by 90 degrees but also reflected.
I have no answer to this. It's just how the math ended up working out.
Maybe it's the implementation of `glm::eulerAngleY` or the 2d pathfinding space is offset?